Shopify/cli
# Remove PHP template from CLI

### WHY are these changes introduced?
* Remix is the green path for app development at Shopify. The PHP template has been hidden from the CLI for several years, now is the appropriate time to remove it.

### WHAT is this pull request doing?

Before
<img width="1384" alt="image" src="https://github.com/user-attachments/assets/c41f6fc2-86b1-4fb5-80f4-8e14e66e6eb9">

After
<img width="1377" alt="image" src="https://github.com/user-attachments/assets/2c9a8fc0-07b8-42a7-9016-63a94f89c0f0">

### How to test your changes?
* Attempt to create an app with the following command. Verify that the PHP template is no longer available.
```
pnpm create-app --template=php
```

* Please verify no where else needs to be updated to remove PHP template from the CLI

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
